src/sizzle.js

```diff
@@ -8,1150 +8,1198 @@
-			subWindow.addEventListener( "unload", unloadHandler, false );
+			// Support: IE 11, Edge
+			if ( subWindow.addEventListener ) {
+				subWindow.addEventListener( "unload", unloadHandler, false );
 
-		// Support: IE 9 - 10 only
-		} else if ( subWindow.attachEvent ) {
-			subWindow.attachEvent( "onunload", unloadHandler );
+				// Support: IE 9 - 10 only
+			} else if ( subWindow.attachEvent ) {
+				subWindow.attachEvent( "onunload", unloadHandler );
+			}
 		}
-	}
 
-	/* Attributes
+		/* Attributes
 	---------------------------------------------------------------------- */
 
-	// Support: IE<8
-	// Verify that getAttribute really returns attributes and not properties
-	// (excepting IE8 booleans)
-	support.attributes = assert(function( el ) {
-		el.className = "i";
-		return !el.getAttribute("className");
-	});
+		// Support: IE<8
+		// Verify that getAttribute really returns attributes and not properties
+		// (excepting IE8 booleans)
+		support.attributes = assert( function( el ) {
+			el.className = "i";
+			return !el.getAttribute( "className" );
+		} );
 
-	/* getElement(s)By*
+		/* getElement(s)By*
 	---------------------------------------------------------------------- */
 
-	// Check if getElementsByTagName("*") returns only elements
-	support.getElementsByTagName = assert(function( el ) {
-		el.appendChild( document.createComment("") );
-		return !el.getElementsByTagName("*").length;
-	});
-
-	// Support: IE<9
-	support.getElementsByClassName = rnative.test( document.getElementsByClassName );
-
-	// Support: IE<10
-	// Check if getElementById returns elements by name
-	// The broken getElementById methods don't pick up programmatically-set names,
-	// so use a roundabout getElementsByName test
-	support.getById = assert(function( el ) {
-		docElem.appendChild( el ).id = expando;
-		return !document.getElementsByName || !document.getElementsByName( expando ).length;
-	});
-
-	// ID filter and find
-	if ( support.getById ) {
-		Expr.filter["ID"] = function( id ) {
-			var attrId = id.replace( runescape, funescape );
-			return function( elem ) {
-				return elem.getAttribute("id") === attrId;
+		// Check if getElementsByTagName("*") returns only elements
+		support.getElementsByTagName = assert( function( el ) {
+			el.appendChild( document.createComment( "" ) );
+			return !el.getElementsByTagName( "*" ).length;
+		} );
+
+		// Support: IE<9
+		support.getElementsByClassName = rnative.test( document.getElementsByClassName );
+
+		// Support: IE<10
+		// Check if getElementById returns elements by name
+		// The broken getElementById methods don't pick up programmatically-set names,
+		// so use a roundabout getElementsByName test
+		support.getById = assert( function( el ) {
+			docElem.appendChild( el ).id = expando;
+			return !document.getElementsByName || !document.getElementsByName( expando ).length;
+		} );
+
+		// ID filter and find
+		if ( support.getById ) {
+			Expr.filter.ID = function( id ) {
```

Comment:
I think we were using quoted names here so that Google Closure Compiler doesn't mangle those names. If that's the case, we can't change them.

@gibson042 Can you confirm or deny?

Reply:
@JuanMaRuiz I consulted this case with the team. We're not sure if that workaround is needed but just in case please don't change array accesses to this new format. You can disable the rule that enforces that in the ESLint config override for `sizzle.js`.